.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+<!--
+    Hello! Thanks for submitting a pull request to MusicBrainz Server.
+    We appreciate your time and interest in helping our project!
+
+    Please use this template to help us review your change.
+
+    Depending on your change, some sections may be unneeded, just remove these.
+    For example, small pull requests usually don’t need the section “Summary”.
+
+    Remember that the more helpful info your pull request includes,
+    the easier it is for us to understand and review your changes.
+
+    Ensure that you’ve read through and followed the Contributing Guidelines, at
+    https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md
+-->
+
+# Summary
+<!--
+    Just give us an overview of your change; don’t go into technical details yet.
+-->
+
+:beginner: Describe this change in 1-2 sentences.
```

Comment:
We've been using a similar Summary field in LB pull request templates and I've felt that it isn't specifically very different from the Pull request title in most cases. I'd be interested in seeing if that happens here too.

Reply:
That goes along with https://github.com/metabrainz/musicbrainz-server/pull/1345#discussion_r381481789. Dropped with https://github.com/metabrainz/musicbrainz-server/pull/1345/commits/1d1bda4deed9ff96af16123c394449dd4cc456ff.